BattleshipConsoleGame/BattleshipConsoleGame/Program.cs

```diff
@@ -4,350 +4,371 @@
 
 namespace BattleshipGame
 {
-    class Program
+class Program
+{
+    public static void Main(string[] args)
     {
-        public static void Main(string[] args)
+        Game game = new Game();
+        Printer.DisplayStartMenu();
+        while (!game.GameOver())
         {
-            int gridWidth = 11;
-            int gridHeight = 11;  //one more than standard size of a Battleship playing field to account for labels for the rows and columns on the user board. variables are used for the rest of the program, so if the size needs changed, it can be done here
+            Console.Clear();
+            game.DisplayUserBoard();
+            UserCoordinates userTarget = GetUserTarget();
+            game.ProcessUserTarget(userTarget);
+        }
+        Printer.DisplayWin();
+    }
 
-            DisplayUserMenu();
+    static UserCoordinates GetUserTarget()
```

Comment:
GetUserTarget() was returning a list of UserCoordinates. That seemed unnecessary, so I changed it to just return a single UserCoordinates object